tests/test_cli/test_add/test_skill.py

```diff
@@ -485,3 +487,23 @@ def teardown_class(cls):
             shutil.rmtree(cls.t)
         except (OSError, IOError):
             pass
+
+
+class TestAddSkillWithContractsDeps(AEATestCase):
+    """Test add skill with contract dependencies."""
+
+    def test_add_skill_with_contracts_positive(self):
+        """Test add skill with contract dependencies positive result."""
+        self.initialize_aea()
+        agent_name = "my_first_agent"
+        self.create_agents(agent_name)
+
+        agent_dir_path = os.path.join(self.t, agent_name)
+        os.chdir(agent_dir_path)
+
+        skill_name = "erc1155_client"
+        self.add_item("skill", "fetchai/{}:0.1.0".format(skill_name))
+
+        skills_path = os.path.join(agent_dir_path, "vendor", FETCHAI_NAME, "skills")
+        skills_folders = os.listdir(skills_path)
```

Comment:
What we are interested in testing is not whether the skill is there, the question is whether its dependency the contract is there...

Reply:
Exactly! Fixed.